modules/serverless-connect-sink-source-odc.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+//  * serverless/channels/connect-channels-event-source.adoc
+// * serverless/event_sources/srvls-connecting-source-sinks.adoc
+
+[id="serverless-connect-sink-source-odc_{context}"]
+= Connect an event source to a sink using the Developer perspective
+
+You can create multiple event source types in {product-title} that can be connected to sinks.
+
+A sink is an addressable resource that can receive incoming events from other Knative Eventing resources. Knative services, channels, and brokers are all examples of sinks.
+// Add functions as services to the list for tech preview
+
+.Prerequisites
+
+To connect an event source to a sink using the *Developer* perspective, ensure that:
+
+* The {ServerlessOperatorName}, Knative Serving, and Knative Eventing are installed on your {product-title} cluster.
+* You have logged in to the web console.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+* You have created a sink.
+
+.Procedure
+
+. Create an event source of any type, by navigating to *+Add* -> *Event Sources* in the *Developer* perspective, and then selecting the event source type that you want to create.
```

Comment:
Per style guide, do not include punctuation in a UI element label. Usually the example is "Click **Save As...**" but I think it might apply to the `+` in **+Add** as well?

Reply:
*+Add* is still used throughout the docs as far as I know for ODC: https://docs.openshift.com/container-platform/4.7/applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.html#odc-creating-sample-applications_odc-creating-applications-using-developer-perspective